code/game/machinery/teleporter.dm

```diff
@@ -290,13 +290,27 @@
 		return
 	return ..()
 
+/obj/machinery/teleport/hub/proc/blockExiles(atom/movable/AM)
```

Comment:
This check is bad - players can simply go into a locker or something else to avoid this